From 5d2626f6e743e76b18272ec5e6bf4ee7e46cfee4 Mon Sep 17 00:00:00 2001
From: John Ferlan <jferlan@redhat.com>
Date: Thu, 8 Aug 2013 15:50:29 -0400
Subject: [PATCH 07/48] spec: Fix capitalization for version check

Following seen on rpm -ivh install:

Preparing...                          ################################# [100%]
Updating / installing...
   1:libvirt-cim-0.6.3-1.fc19         ################################# [100%]
   /var/tmp/rpm-tmp.uY5oGu: line 5: /etc/init.d/tog-pegasus: No such file or directory

Turns out commit id '8480cc79' inadvertently capitalized Fedora when replacing
a check that got removed by commit id '07adabc2'.

"Fedora" shouldn't have been capitalized in the check to use
systemctl instead of /etc/init.d/tog-pegasus

Signed-off-by: John Ferlan <jferlan@redhat.com>
---
 libvirt-cim.spec.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libvirt-cim.spec.in b/libvirt-cim.spec.in
index 98f2a75..459650c 100644
--- a/libvirt-cim.spec.in
+++ b/libvirt-cim.spec.in
@@ -96,7 +96,7 @@ rm -fr $RPM_BUILD_ROOT
 
 %{_datadir}/%{name}/install_base_schema.sh %{_datadir}/%{name}
 
-%if 0%{?Fedora} >= 17 || 0%{?rhel} >= 7
+%if 0%{?fedora} >= 17 || 0%{?rhel} >= 7
     if [ "`systemctl is-active tog-pegasus.service`" = "active" ]
     then
         systemctl restart tog-pegasus.service
-- 
1.8.5.3

